neuralop/layers/normalization_layers.py

```diff
@@ -28,3 +28,26 @@ def forward(self, x):
         weight, bias = torch.split(self.mlp(self.embedding), self.in_channels, dim=0)
 
         return nn.functional.group_norm(x, self.in_channels, weight, bias, eps=self.eps)
+
+class FlattenedInstanceNorm1d(nn.Module):
+    def __init__(self, num_features: int):
```

Comment:
Should we allow for the additional `kwargs` of InstanceNorm1d, in particular `affine` and `eps`?

Reply:
But I think we can also just directly use `torch.nn.functional.instance_norm` (since we anyways take care that the shape is correct and that seems to be the only difference)

Reply:
I updated the structure to pass in the additional `kwargs`.